internal/users/data/mfa-factors.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "email": "antoinette@contoso.com",
```

Comment:
Same thing as above, I think we could keep just one entry here so it's easier to edit.

Reply:
I think each example is useful, per the user import schema docs -> https://auth0.com/docs/manage-users/user-migration/bulk-user-import-database-schema-and-examples#examples